src/sagemaker/multidatamodel.py

```diff
@@ -121,7 +121,11 @@ def __init__(
             )
 
     def prepare_container_def(
-        self, instance_type=None, accelerator_type=None, serverless_inference_config=None
+        self,
+        instance_type=None,
+        accelerator_type=None,
+        serverless_inference_config=None,
+        accept_eula=None,
```

Comment:
Another plus to adhering to `getattr` is now you won't need to make every modification here